src/app/routes/index.js

```diff
@@ -5,14 +5,25 @@ import ErrorPage from '../containers/Error';
 import getArticleInitialData from './getInitialData/article';
 import getFrontpageInitialData from './getInitialData/frontpage';
 import getMediaPageInitialData from './getInitialData/mediapage';
-import getMediaAssetPageInitialData from './getInitialData/mediaAssetPage';
+import getCpsAssetInitialData from './getInitialData/cpsAsset';
 import {
   articleRegexPath,
   frontpageRegexPath,
   mediaRadioAndTvRegexPathsArray,
-  mediaAssetPageRegexPath,
+  cpsAssetPageRegexPath,
 } from './regex';
 
+const pages = {
+  MAP: MediaPage,
+  FIX: FrontPage,
+};
+
+const CpsAsset = props => {
```

Comment:
IMO this was better the first time, in its own container

Reply:
I thought it would be better to keep it in the routes files because it contains logic specific to routing.